middleware/src/middleware.go

```diff
@@ -0,0 +1,105 @@
+package middleware
+
+import (
+	"time"
+	//"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/rpcclient"
+	//"github.com/btcsuite/btcutil"
+	"github.com/fiatjaf/lightningd-gjson-rpc"
+	//"github.com/tidwall/gjson"
+	"github.com/digitalbitbox/bitbox-base/middleware/util/logging"
+	"github.com/sirupsen/logrus"
+)
+
+type MiddlewareInfoStruct struct {
+	Blocks         int64   `json:"blocks"`
+	Difficulty     float64 `json:"difficulty"`
+	LightningAlias string  `json:"alias"`
```

Comment:
Nit: Maybe the exported JSON name also could be `lightning_alias`, to make it less ambiguous which kind of `alias` we're talking about. I.e same as the struct member's name?